project/database/cleanup.py

```
@@ -0,0 +1,209 @@
+        - Grace period for appeals: +60 days
+        - Total: 90 days is conservative and compliant
+        """
+        cutoff_date = datetime.now(timezone.utc) - timedelta(days=days_old)
+
+        try:
+            # Find old soft-deleted warnings
+            old_warnings = (
+                self.db.query(SecureWarning)
+                .filter(
+                    and_(
+                        SecureWarning.is_deleted == True,
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `SecureWarning.is_deleted:` for truth checks

Replace with `SecureWarning.is_deleted`

(E712)